src/modelmanager.hpp

```diff
@@ -176,6 +176,10 @@ class ModelManager {
         return models;
     }
 
+    const ModelConfig& getModelConfig(std::string modelName) {
+        return servedModelConfigs[modelName];
```

Comment:
This will add modelconfig to servedModelConfigs if modelName does not exist, which is not desired